tools/gen_addon_readme.rst.jinja

```diff
@@ -6,7 +6,12 @@
 {{ fragments[name] }}
 {% endif %}
 {%- endmacro -%}
+=======
+.. image:: https://odoo-community.org/web/image/4610395-7254fada/OCA_banner_readme_1256x150px.png
+   :target: https://odoo-community.org/get-involved?ref=k8vx13&utm_source=referral-readme&utm_medium=referral
```

Comment:
What is "ref=k8vx13" ? Also here should we not use a stable URL in case the details of our website change in the future?